backend/api/office_hours/oh_ticket.py

```diff
@@ -0,0 +1,123 @@
+"""OH Ticket API
+
+This API is used to access OH ticket data for history purposes."""
+
+from fastapi import APIRouter, Depends
+
+from backend.models.office_hours.oh_ticket import (
+    OfficeHoursTicket,
+    OfficeHoursTicketDraft,
+)
+from backend.models.office_hours.oh_ticket_details import OfficeHoursTicketDetails
+from ..authentication import registered_user
+from ...models import User
+
+
+__authors__ = ["Sadie Amato", "Bailey DeSouza"]
+__copyright__ = "Copyright 2024"
+__license__ = "MIT"
+
+
+api = APIRouter(prefix="/api/office-hours/ticket")
+openapi_tags = {
+    "name": "Office Hours",
+    "description": "Office hours ticket functionality",
+}
+
+
+@api.post("", response_model=OfficeHoursTicketDetails, tags=["Office Hours"])
+def new_oh_ticket(
+    oh_ticket: OfficeHoursTicketDraft,
+    subject: User = Depends(registered_user),
+    oh_ticket_service: OfficeHoursTicketService = Depends(),
+) -> OfficeHoursTicketDetails:
+    """
+    Adds a new OH ticket to the database
+
+    Returns:
+        OfficeHoursTicketDetails: OH Ticket created
+    """
+    return oh_ticket_service.create(subject, oh_ticket)
+
+
+@api.get(
+    "/{section_id}",
+    response_model=list[OfficeHoursTicketDetails],
+    tags=["Office Hours"],
+)
+def get_oh_tickets_by_section(
+    section_id: int,
```

Comment:
Just a quick thought! Since we are working with both an academic section and an office hours section, maybe we could specify the variable name to be `oh_section_id` to hopefully avoid confusion?

Reply:
Renamed this in recent commit! Plus changed another place where we just wrote `event_id` to be `oh_event_id`